<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveBlurringWidget</class>
 <widget class="QMainWindow" name="InteractiveBlurringWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="gfxviewOriginal"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="gfxviewBlurred"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="FloatSlider" name="sldBlurKernelVariance">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Float slider</string>
      </property>
      <property name="whatsThis">
       <string>The float slider widget allows the user to select a float value.</string>
      </property>
      <property name="name">
       <string>Blur kernel variance</string>
      </property>
      <property name="maxValueText">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabeledSlider</class>
   <extends>QWidget</extends>
   <header>LabeledSlider.h</header>
  </customwidget>
  <customwidget>
   <class>FloatSlider</class>
   <extends>LabeledSlider</extends>
   <header>FloatSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
